code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -714,3 +714,74 @@
 		segment_length = armsy.get_length()
 		qdel(armsy)
 		return
+
+/obj/effect/proc_holder/spell/pointed/void_blink
+	name = "Void Phase"
+	desc = "Let's you blink to your pointed destination, causes 3x3 aoe damage bubble around your pointed destination and your current location"
+	invocation_type = INVOCATION_WHISPER
+	invocation = "RE'L'TY PH'S'E"
+	clothes_req = FALSE
+	range = 9
+	action_background_icon_state = "bg_ecult"
+	charge_max = 300
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "voidblink"
+	selection_type = "range"
+
+/obj/effect/proc_holder/spell/pointed/void_blink/cast(list/targets, mob/user)
+	. = ..()
+	var/target = targets[1]
+	var/turf/targeted_turf = isturf(target) ? target : get_turf(target)
+
+	playsound(user,'sound/magic/voidblink.ogg',100)
+	playsound(targeted_turf,'sound/magic/voidblink.ogg',100)
+
+	new /obj/effect/temp_visual/voidin(user.drop_location())
+	new /obj/effect/temp_visual/voidout(targeted_turf)
+
+	for(var/mob/living/living_mob in range(1,user)-user)
+		living_mob.adjustOxyLoss(30)
+
+	for(var/mob/living/living_mob in range(1,targeted_turf)-user)
+		living_mob.adjustOxyLoss(30)
+
+	do_teleport(user,targeted_turf,channel = TELEPORT_CHANNEL_MAGIC)
+
+
+/obj/effect/temp_visual/voidin
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "void_blink_in"
+	alpha = 150
+	duration = 6
+	pixel_x = -32
+	pixel_y = -32
+
+/obj/effect/temp_visual/voidout
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "void_blink_out"
+	alpha = 150
+	duration = 6
+	pixel_x = -32
+	pixel_y = -32
+
+/obj/effect/proc_holder/spell/targeted/void_pull
+	name = "Void pull"
+	desc = "Call the void, this pulls all nearby people closer to you"
+	invocation_type = INVOCATION_WHISPER
+	invocation = "BR'NG F'RTH TH'M T' M'"
+	clothes_req = FALSE
+	action_background_icon_state = "bg_ecult"
+	range = -1
+	include_user = TRUE
+	charge_max = 400
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "voidpull"
+
+/obj/effect/proc_holder/spell/targeted/void_pull/cast(list/targets, mob/user)
+	. = ..()
+	playsound(user,'sound/magic/voidblink.ogg',100)
+	new /obj/effect/temp_visual/voidin(user.drop_location())
+	for(var/mob/living/livies in view(7,user)-user)
+		for(var/i in 0 to 2)
+			livies.forceMove(get_step_towards(livies,user))
```

Comment:
something to slow them down for a moment or two might also be nice

sort of like the shove movement speed debuff, but not the actual shove movement speed debuff, because then you could disarm someone's gun with a single shove after using this power (because the "two disarms in rapid succession=a gun disarm, regardless of whether or not you're shoving someone against something" thing uses the shove movement speed debuff to keep track of whether or not someone has been shoved recently).

Reply:
it now knockdowns